### ISSUE TITLE
Corrupted Model after training

### ISSUE THREAD

**Issue by CypherpunkSamurai** [victorchall/EveryDream2trainer#73]
I'm running a everydream trainer with configs and it seems to be giving corrupted models.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/diffusers/modeling_utils.py", line 97, in load_state_dict
    return torch.load(checkpoint_file, map_location="cpu")
  File "/usr/local/lib/python3.10/site-packages/torch/serialization.py", line 777, in load
    with _open_zipfile_reader(opened_file) as opened_zipfile:
  File "/usr/local/lib/python3.10/site-packages/torch/serialization.py", line 282, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/diffusers/modeling_utils.py", line 103, in load_state_dict
    if f.read().startswith("version"):
  File "/usr/local/lib/python3.10/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 128: invalid start byte

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/content/EveryDream2trainer/scripts/txt2img.py", line 101, in <module>
    main(args)
  File "/content/EveryDream2trainer/scripts/txt2img.py", line 68, in main
    unet = UNet2DConditionModel.from_pretrained(args.diffusers_path, subfolder="unet")
  File "/usr/local/lib/python3.10/site-packages/diffusers/modeling_utils.py", line 489, in from_pretrained
    state_dict = load_state_dict(model_file)
  File "/usr/local/lib/python3.10/site-packages/diffusers/modeling_utils.py", line 115, in load_state_dict
    raise OSError(
OSError: Unable to load weights from checkpoint file for '/content/EveryDream2trainer/logs/test-ep99-gs01400/ckpts/last-test-ep99-gs01400/unet/diffusion_pytorch_model.bin' at '/content/EveryDream2trainer/logs/test-ep99-gs01400/ckpts/last-test-ep99-gs01400/unet/diffusion_pytorch_model.bin'. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True.
```

Training Command:
```python
python train.py \
--resume_ckpt runwayml/stable-diffusion-v1-5 \
--gradient_checkpointing \
--amp \
--batch_size 6 \
--grad_accum 1 \
--cond_dropout 0.00 \
--data_root EveryDream2trainer/images \
--flip_p 0.00 \
--lr 1e-06 \
--lr_decay_steps 0 \
--lr_scheduler constant \
--lr_warmup_steps 0 \
--max_epochs 100 \
--project_name test \
--resolution 512 \
--sample_prompts sample_prompts.txt \
--sample_steps 300 \
--save_every_n_epoch 0 \
--seed 555 \
--shuffle_tags \
--useadam8bit
```

**Comment by victorchall** [comment]
Discussed on the discord, but I ran a test and no problems.  The error is in inference after something already went wrong saving the files and they are corrupt, maybe process was killed?  I would look at your output on the training cell to spot the actual problem, its too late at this point to fix the corrupt files.

**Comment by CypherpunkSamurai** [comment]
I checked the logs and the training had completed successfully, and the files also got copied to two remotes. after training (checkpoint and diffuser)

The training was running on a P100. I tried training on a T4 and a P100 and turns out in T4 trained model are of right sizes, where's on a P100 the trained models were half the original sizes. The training notebook and configs were copied in both cases.

I also reverted back to commit `27b66ab54f9d2515211a373c7f3127d5b66b4921` with `git reset --hard` and the trained model worked fine on the P100.

Not sure what's happening here you might wanna tag this "additional information required" for now. I'll try running tests again / providing you the notebooks.
